hax-lib/src/int/mod.rs

```diff
@@ -86,6 +87,10 @@ impl Int {
         Self::new(num_bigint::BigInt::from_str(s).unwrap())
     }
 
+    pub fn _unsafe_from_hex(s: &str) -> Self {
+        Self::new(num_bigint::BigInt::from_str_radix(s, 16).unwrap())
+    }
+
```

Comment:
Ah, the `_unsafe_from_str` method is treated specially by the engine, and this should not be used directly.
We have troubles with the documentation of hax-lib, so it's hard to discover. You can run `RUSTCFLAGS="--cfg hax" cargo doc` on the `hax-lib` folder to get the proper documentation.

But basically, here you should use the `int!` macro, which, unfortunately, doesn't supports hex. I opened issue https://github.com/cryspen/hax/issues/1532.

Let me fix this issue so that you can get rid of this.

Reply:
This is merged in main now, could you drop `_unsafe_from_hex` and use the `int` macro now?

Reply:
Yes, that seem to work, thanks a lot for the quick change! I will take a look at the locally generated docs as you suggested to see if I can find more stuff that makes sense for me to use. Thanks a lot for the hint!